src/createServer.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+
+const { router: userRouter } = require('./routes/user.route');
+const { router: expenseRouter } = require('./routes/expense.route');
+const { clear: clearUsers } = require('./services/user.service');
+const { clear: clearExpenses } = require('./services/expense.service');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  clearUsers();
+  clearExpenses();
+
+  app.use('/users', express.json(), userRouter);
+  app.use('/expenses', express.json(), expenseRouter);
```

Comment:
```suggestion
  app.use(express.json());
  app.use('/users', userRouter);
  app.use('/expenses', expenseRouter);
```
You can apply `express.json()` for all routes